legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -118,7 +119,12 @@ def post_businesses():
     # @TODO rollback bootstrap if there is A failure, awaiting changes in the affiliation service
     bootstrap = RegistrationBootstrapService.create_bootstrap(filing_account_id)
     if not isinstance(bootstrap, RegistrationBootstrap):
-        return {'error': babel('Unable to create {0} Filing.'.format(Filing.FILINGS[filing_type]['title']))}, \
+        if json_input['filing'][filing_type] == Filing.FILINGS['amalgamation']['name']:
+            amalgamation_type = json_input['filing'][filing_type]['amalgamationType']
+            title = Filing.FILINGS[filing_type][amalgamation_type]['title']
+        else:
+            title = Filing.FILINGS[filing_type]['title']
+        return {'error': babel('Unable to create {0} Filing.'.format(title))}, \
```

Comment:
Can we update this to use [get_filings_sub_type](https://github.com/bcgov/lear/blob/c2e4af61374e77f489708d0da67e3ba4692c8f79/legal-api/src/legal_api/models/filing.py#L898)?  

You'll  need to update the [dictionary](https://github.com/bcgov/lear/blob/c2e4af61374e77f489708d0da67e3ba4692c8f79/legal-api/src/legal_api/models/filing.py#L314-L319) here.  

I think we can assume that the filing sub-type will be like restorations.  i.e. `type` will be the key used in the schema to determine the type of amalgamation

Reply:
I updated it as `type`